os/tools/mkbootparam.py

```diff
@@ -24,24 +24,30 @@
 bootparam_file_path = os.path.dirname(__file__) + '/../../build/output/bin/bootparam.bin'
 user_file_dir = os.path.dirname(__file__) + '/../../build/output/bin/user'
 
-SIZE_OF_BOOTPARAM_PART = 8192
+SIZE_OF_BPx = 4096
+
+# The partition size of dual BPs
+SIZE_OF_BP_PARTITION = SIZE_OF_BPx * 2
 
 ###########################################################################################
 #
-# This script generates Boot Parameters for kernel.
+# This script generates Boot Parameters.
+#
+# Boot Parameter works based on dual partition, BP1 and BP2.
+# It puts BP data to first BP, BP1 and clears second BP, BP2.
 #
-# Boot Parameter information :
+# Boot Parameter information (BPx) :
 #  - Boot Parameter Format Version 1 (bootparam_format_version = 1)
 #  - The data from 'App Count' depends on CONFIG_APP_SEPARATION.
 # +------------------------------------------------------------------------------------------
 # | Checksum | BP Version | BP Format Version | Active Idx | First Address | Second Address |
 # | (4bytes) |  (4bytes)  |      (4bytes)     |   (1byte)  |    (4bytes)   |    (4bytes)    |
 # +------------------------------------------------------------------------------------------
-# ----------------------------------------------------------------------------------------------------------------+
-# | App Count | App1 Name  | App1 Active Idx | App2 Name  | App2 Active Idx |     App# ...   |      Reserved      |
-# |  (1byte)  | (16 bytes) |   (1 bytes)     | (16 bytes) |    (1 bytes)    |  (## bytes)    |(8Kbytes - 21bytes) |
-# ----------------------------------------------------------------------------------------------------------------+
-#  * The Boot Parameters (8KB) should be downloaded and be located at the end of a flash.
+# ----------------------------------------------------------------------------------------------------------------------+
+# | App Count | App1 Name  | App1 Active Idx | App2 Name  | App2 Active Idx |     App# ...   |         Reserved         |
+# |  (1byte)  | (16 bytes) |   (1 bytes)     | (16 bytes) |    (1 bytes)    |  (## bytes)    |(4Kbytes - (21 + @)bytes) |
+# ----------------------------------------------------------------------------------------------------------------------+
+#  * The Boot Parameters, BP1 and BP2 (8KB) should be downloaded and be located at the end of a flash.
```

Comment:
User could understand two bins (BP1 and BP2) should be downloaded. Could you rewrite it clearly?